docs/source/technical_tutorials/visualization/visualize_lonboard.ipynb

```diff
@@ -0,0 +1,574 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    search = catalog.search(collections=[collection_id], max_items=5)
</pre><p>do we need to search for items if we hardcode a specific item in the "Load Data" step? if you still want to utilize the STAC catalog, get_items(*ids) would be an option</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>